docs/INSTALL.md

```diff
@@ -1,8 +1,7 @@
 Getting started
 ===============
 
-This document describes how to manually configure your system for running
-CyclOSM.
+This document describes how to manually configure your system for running CyclOSM. If you prefer quick, platform independent setup for a development environment, without the need to install and configure tools by hand, follow a Docker installation guide in [DOCKER.md](https://github.com/cyclosm/cyclosm-cartocss-style/blob/master/docs/DOCKER.md).
```

Comment:
I would mention that preferred method is running CyclOSM manually. Docker alternative is provided as an alternative.

(reason being @Florimondable and I are mainly using the manual approach, so Docker approach is more community supported, although we'll do our best to update everything as needed)